evalml/data_checks/id_columns_data_check.py

```diff
@@ -89,66 +89,81 @@ def validate(self, X, y=None):
             ...     }
             ... ]
 
-            If the first column of the dataframe is identified as an ID column it is most likely the primary key.
+            Despite being all unique, "Country_Rank" will not be identified as an ID column as id_threshold is set to 1.0
+            by default and its name doesn't indicate that it's an ID.
 
             >>> df = pd.DataFrame({
-            ...     "sales_id": [0, 1, 2, 3, 4],
-            ...     "customer_id": [123, 124, 125, 126, 127],
-            ...     "Sales": [10, 42, 31, 51, 61]
+            ...    "humidity": ["high", "very high", "low", "low", "high"],
+            ...    "Country_Rank": [1, 2, 3, 4, 5],
+            ...    "Sales": ["very high", "high", "high", "medium", "very low"]
             ... })
             ...
             >>> id_col_check = IDColumnsDataCheck()
+            >>> assert id_col_check.validate(df) == []
+
+            However lowering the threshold will cause this column to be identified as an ID.
+
+            >>> id_col_check = IDColumnsDataCheck()
+            >>> id_col_check = IDColumnsDataCheck(id_threshold=0.95)
             >>> assert id_col_check.validate(df) == [
             ...     {
-            ...         "message": "The first column 'sales_id' is most likely to be the ID column. Columns 'customer_id' are also 100.0% or more likely to be an ID column",
+            ...         "message": "Columns 'Country_Rank' are 95.0% or more likely to be an ID column",
             ...         "data_check_name": "IDColumnsDataCheck",
             ...         "level": "warning",
-            ...         "code": "HAS_ID_FIRST_COLUMN",
-            ...         "details": {'columns': ["sales_id", "customer_id"], 'rows': None},
+            ...         "details": {"columns": ["Country_Rank"], "rows": None},
+            ...         "code": "HAS_ID_COLUMN",
             ...         "action_options": [
             ...             {
-            ...                 "code": "SET_FIRST_COL_ID",
+            ...                 "code": "DROP_COL",
             ...                 "data_check_name": "IDColumnsDataCheck",
             ...                 "parameters": {},
-            ...                 "metadata": {'columns': ["sales_id", "customer_id"], 'rows': None}
+            ...                 "metadata": {"columns": ["Country_Rank"], "rows": None}
             ...             }
             ...         ]
-            ...    }
+            ...     }
             ... ]
 
-            Despite being all unique, "Country_Rank" will not be identified as an ID column as id_threshold is set to 1.0
-            by default and its name doesn't indicate that it's an ID.
+            If the first column of the dataframe is 100% likely to be an ID column, it is probably the primary key.
```

Comment:
Nit: "is 100% likely" doesn't make much sense. Can you rephrase this to something along the lines of "has all unique values" instead?